lib/systems/include/BaseLaunchController.h

```diff
@@ -4,6 +4,11 @@
 #include "BaseController.h"
 #include <algorithm>
 #include <math.h> 
+
+/// @brief Modes to define launch behavior, where each one waits for acceleration request threshold to move to next mode
```

Comment:
would be nice if there was the exact flow of the launch state machine:

what state do we start in and what exactly transitions between the states and what are all the possible state transitions?